internal/pkg/task/env_runner.go

```diff
@@ -135,3 +140,21 @@ func (r *EnvRunner) validateDependencies() error {
 
 	return nil
 }
+
+func appendUniqueStrings(s1 []string, s2 ...string) []string {
```

Comment:
this is totally fine, in case you're curious Go has also a little entry on how to achieve it: https://github.com/golang/go/wiki/SliceTricks#in-place-deduplicate-comparable

I'd lean to that implementation, but this is totally fine too!

Reply:
that's a good page to bookmark, thanks!